yelp_package/dockerfiles/trusty/Dockerfile

```diff
@@ -20,6 +20,11 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN echo "deb http://repos.mesosphere.com/ubuntu trusty main" > /etc/apt/sources.list.d/mesosphere.list && \
     apt-key adv --keyserver keyserver.ubuntu.com --recv 81026D0004C44CF7EF55ADF8DF7D54CBE56151BF
 
+# Need Python 3.6
+RUN apt-get update > /dev/null
+RUN apt-get install -y --no-install-recommends software-properties-common
+RUN add-apt-repository ppa:fkrull/deadsnakes
```

Comment:
these should all be in a single `RUN` line, otherwise docker caching could give you an outdated `apt-get update` result